getting-started/first-steps.md

```diff
@@ -0,0 +1,93 @@
+---
+title: First Steps
+page_title: First Steps
+description: First Steps
+slug: getting-started-first-steps
+tags: get,started,first,steps
+published: True
+position: 0
+---
+
+# First Steps
+
+This article explains how to get the Telerik UI for {{ site.framework_name }} controls in your project and start using them quickly.
+
+Once you have your first simple control up and running, take a look at the [next steps](#next-steps) section to start exploring the control functionality in more detail.
+
+## Download the Controls
+
+The easiest way to get the controls to your development machine is to use the [Progress Control Panel](https://www.telerik.com/download-trial-file/v2/control-panel) or to download the [automated MSI installer]({%slug installation-installing-from-msi-wpf%}) from [your telerik.com account](https://www.telerik.com/account/product-download?product=RCWPF).
+
+>caption Figure 1: Download automated (.msi) installer
+
+![Download automated installer Telerik_UI_for_WPF_<version>_Dev.msi](images/download-msi.png "Telerik_UI_for_WPF_<version>_Dev.msi")
+
+> If you are not a customer, you can [download a free, fully functional trial](https://www.telerik.com/download-trial-file/v2-b/ui-for-wpf) and the same options will apply to you as well.
+
+>tip The following article can help you choose the installation type that is most suitable for your needs and preferences: [Which File Do I Need to Install]({%slug installation-installing-which-file-do-i-need%}).
+
+## Create a UI for {{ site.framework_name }} Project
+
+The easiest way to create a Telerik UI for {{ site.framework_name }} project is to use **Visual Studio Extensions** which are distributed with the Telerik UI for {{ site.framework_name }} installer.
+
+The Visual Studio Extensions can be accessed through the Telerik | Telerik UI for {{ site.framework_name }} menu which has different menu items depending on the selected project in Visual Studio. The extensions can be accessed through the context menu of a {{ site.framework_name }} Application as well.
+
+You can then create the Telerik UI for {{ site.framework_name }} project in one of the 3 following ways:
+
+* If you **do not have a project**, run the [Create Project Wizard]({%slug radcontrols-for-wpf-vs-extensions-project-creation%}).{% if site.framework_name == 'WPF' %} You can either create a new blank project or pick one of the available [predefined templates]({%slug visual-studio-templates%}).{% endif %}
```

Comment:
This article seems like it was created only for WPF. In this case you don't need the 'if' statement.